_studio/mfx_lib/encode_hw/h264/include/mfx_h264_encode_hw_utils.h

```diff
@@ -998,6 +998,7 @@ namespace MfxHwH264Encode
             , m_LtrQp(0)
             , m_RefOrder(-1)
             , m_RefQp(0)
+            , m_bFrameReady(0)
```

Comment:
Maybe it's better to set it t "false" directly instead of "0" ?

Reply:
I agree

Reply:
@DenWolf : please, take a look again and approve PR if you are fine with the changes.